src/lib/management/pages/StudentsPage/StudentsManagementTable.js

```diff
@@ -1,6 +1,15 @@
-import React from 'react';
```

Comment:
this file is too large
I recommend dividing it to ControlPanel (name it as you think) and ControlTable
See which variables overlap with both, pull them in the father and pass them